iac/terraform/aws/budget.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_budgets_budget" "blood_connect_budget" {
+  count = (var.budget_settings.set_budget ? 1 : 0)
```

Comment:
count variables like this should exist in a seperate file so that we can make self commenting code, like
```hcl
locals {
    is_budget_set = var.budget_settings.set_budget ? 1 : 0
}
```